app/views/admin/loans/content/_breakeven_charts.html.slim

```diff
@@ -0,0 +1,24 @@
+.summary-chart.breakeven-chart.row
+  h3.col-sm-12 = t("loan.breakeven.charts_title")
+  .chart.col-lg-6.col-sm-12
```

Comment:
Thanks for designing for mobile! I'm pretty sure that `col-sm-12` is unnecessary though, as blocks will default to full width if unspecified.